### PR TITLE
Annotations of methods corrected

### PR DESCRIPTION
**Pull request by quotidianus**

### CODE REVIEW

File: src/Entity/OuterExtension/HasEmail.php [hunk 1]
```diff
@@ -24,7 +24,7 @@ public function getEmail()
      *
      * @param object $email
      *
-     * @return Object
+     * @return object
```
Comment (PapsOu) [inline src/Entity/OuterExtension/HasEmail.php]: The returned type should be « $this » because of fluent accessors

**Review by marcoslibre — APPROVED** [review]

File: src/Entity/OuterExtension/HasEmail.php [hunk 2]
```diff
@@ -24,7 +24,7 @@ public function getEmail()
      *
      * @param object $email
```
Comment (PapsOu) [inline src/Entity/OuterExtension/HasEmail.php]: param type should be « Email » ?

Comment (PapsOu) [reply src/Entity/OuterExtension/HasEmail.php]: instead of object..

**Review by FanchTheSystem — APPROVED** [review]

**Review by PapsOu — COMMENTED** [review]
Please check these 3 comments

File: src/Entity/OuterExtension/HasEmail.php (hunk 2 above)
Comment (quotidianus) [reply src/Entity/OuterExtension/HasEmail.php]: Ok. Corrected and pushed again.

File: src/Entity/OuterExtension/HasEmail.php (hunk 1 above)
Comment (quotidianus) [reply src/Entity/OuterExtension/HasEmail.php]: Ok. Corrected and pushed again.

File: src/Entity/OuterExtension/HasEmail.php [hunk 3]
```diff
@@ -22,9 +22,9 @@ public function getEmail()
     /**
      * Set email
      *
-     * @param object $email
+     * @param Email
```
Comment (PapsOu) [inline src/Entity/OuterExtension/HasEmail.php]: missing var name :

@param Email $email

Comment (quotidianus) [reply src/Entity/OuterExtension/HasEmail.php]: Ah OK. Precisely, it has seemed me strange to do not indicate it. Pushed done again.

**Review by PapsOu — APPROVED** [review]

**Pull request merged**